spec.html

```diff
@@ -787,68 +787,98 @@ <h1>Throw an Exception</h1>
+          1. *[before]* _argument_ *[after]*
         </emu-alg>
-        <p>Algorithms steps that say or are otherwise equivalent to:</p>
+        <p>where</p>
+        <ul>
+          <li>*[before]*, if present, is any algorithm text that does not contain ReturnIfAbrupt.
+          <li>*[after]*, if present, is any algorithm text.
+          <li>_argument_ is any alias.
+        </ul>
+
+        <p>An algorithm step that says:</p>
         <emu-alg>
-          1. ReturnIfAbrupt(AbstractOperation()).
+          1. *[before]* ReturnIfAbrupt(AbstractOperation(*[arguments]*)) *[after]*
         </emu-alg>
-        <p>mean the same thing as:</p>
+        <p>is equivalent to the sequence of steps:</p>
         <emu-alg>
-          1. Let _hygienicTemp_ be AbstractOperation().
+          1. Let _hygienicTemp_ be AbstractOperation(*[arguments]*).
           1. If _hygienicTemp_ is an abrupt completion, return _hygienicTemp_.
           1. Else if _hygienicTemp_ is a Completion Record, set _hygienicTemp_ to _hygienicTemp_.[[Value]].
+          1. *[before]* _hygienicTemp_ *[after]*
         </emu-alg>
-        <p>Where _hygienicTemp_ is ephemeral and visible only in the steps pertaining to ReturnIfAbrupt.</p>
-        <p>Algorithms steps that say or are otherwise equivalent to:</p>
+        <p>where</p>
+        <ul>
+          <li>*[before]*, if present, is any algorithm text that does not contain ReturnIfAbrupt.
+          <li>*[after]*, if present, is any algorithm text.
+          <li>*[arguments]*, if present, is any algorithm text that provides an argument list to AbstractOperation.
+          <li>AbstractOperation is any abstract operation reference, regardless of the notation used to refer to it.
+          <li>_hygienicTemp_ is ephemeral and visible only in the steps pertaining to ReturnIfAbrupt.
+        </ul>
+
+        <p>For example, an algorithm step that says:</p>
         <emu-alg>
-          1. Let _result_ be AbstractOperation(ReturnIfAbrupt(_argument_)).
+          1. Let _result_ be _someValue_ if ReturnIfAbrupt(_someValue_.OperationName(_firstArgument_, _secondArgument_)) is *true*, and _someOtherValue_ otherwise.
         </emu-alg>
-        <p>mean the same thing as:</p>
+        <p>is equivalent to the sequence of steps:</p>
         <emu-alg>
-          1. If _argument_ is an abrupt completion, return _argument_.
-          1. If _argument_ is a Completion Record, set _argument_ to _argument_.[[Value]].
-          1. Let _result_ be AbstractOperation(_argument_).
+          1. Let _hygienicTemp_ be _someValue_.OperationName(_firstArgument_, _secondArgument_).
+          1. If _hygienicTemp_ is an abrupt completion, return _hygienicTemp_.
+          1. Else if _hygienicTemp_ is a Completion Record, set _hygienicTemp_ to _hygienicTemp_.[[Value]].
+          1. Let _result_ be _someValue_ if _hygienicTemp_ is *true*, and _someOtherValue_ otherwise.
```

Comment:
```suggestion
          1. If _hygienicTemp_ is *true*, let _result_ be _someValue_; else let _result_ be _someOtherValue_.
```
(Similar to @ljharb's suggestion at L829.)